cypress/e2e/signIn.cy.js

```diff
@@ -12,23 +17,40 @@ describe('Sign In page', () => {
 
   before(() => {
     cy.task('db:clear');
+  });
+
+  beforeEach(() => {
     cy.task('generateUser').then((generateUser) => {
       user = generateUser;
+      signInPage.visit();
+      cy.register(user.email, user.username, user.password);
     });
   });
 
   it('should provide an ability to log in with existing credentials', () => {
-    signInPage.visit();
-    cy.register(user.email, user.username, user.password);
-
-    signInPage.typeEmail(user.email);
-    signInPage.typePassword(user.password);
+    signInPage.fillLoginCredentials(user.email, user.password);
```

Comment:
The 'fillLoginCredentials' function is a good example of a pure function. However, it is not clear what this function does from the name. Try to be more explicit with function names.

Reply:
Done
New name is 'fillLoginFieldsOnCredentials'